databus-core/databus-core-impl/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
   compile externalDependency.json
   compile externalDependency.log4j
   compile externalDependency.netty
+  compile 'com.mchange:c3p0:0.9.5'
```

Comment:
Can you move the dependency definition to subprojects.gradle in the "ext.externalDependency" array:

'c3p0': 'com.mchange:c3p0:0.9.5'

In this build.gradle, you add 

compile externalDependency.c3p0